### ISSUE TITLE
Error running using redis CLI

### ISSUE THREAD

**Issue by vanshavenger** [DiceDB/dice#1207]
## Steps to reproduce

redis-cli -p 7379
## Expected output

The expected output when the above set of commands (maybe when run on Redis)

```
Should run as expected like earlier
```

## Observed output

The observed output when the above set of commands when run on DiceDB

<img width="914" alt="Screenshot 2024-10-27 at 8 49 34 PM" src="https://github.com/user-attachments/assets/128e3ee2-e968-4d73-8a94-35e4457df443">

**Comment by shashi-sah2003** [comment]
@JyotinderSingh @lucifercr07 this is happening due to https://github.com/DiceDB/dice/pull/993 
pls revert it to fix this

**Comment by shashi-sah2003** [comment]
@lucifercr07 can you pls revert #993 back as it is still creating issue at local environment running the dicedb server?

**Comment by tejasa97** [comment]
do we have a fix for this yet?
i'm still unable to run this locally with
```
redis-cli -p 7379
```

**Comment by lucifercr07** [comment]
@tejasa97 can you please try with https://github.com/DiceDB/dicedb-cli, meanwhile will look into this.

**Comment by tejasa97** [comment]
will do. thanks

**Comment by apoorvyadav1111** [comment]
Hi @tejasa97, @shashi-sah2003, for know if dicedb-cli does not work for you, you can comment out the following lines in eval.go.
```go
	// case Docs:
	// 	return evalCommandDocs(args[1:])
```

**Comment by shashi-sah2003** [comment]
> Hi @tejasa97, @shashi-sah2003, for now if dicedb-cli does not work for you, you can comment out the following lines in eval.go.
> 
> ```go
> 	case Docs:
>  	       return evalCommandDocs(args[1:])
> ```

yeah it's working now

**Comment by lucifercr07** [comment]
Closing, merged as part of #1312
